app/utils/url_utils.rb

```diff
@@ -43,4 +43,13 @@ def extract_locale_from_url(url)
   def strip_port_from_host(host)
     host.split(":").first
   end
+
+  # Naive join method, which can be used to normalize multiple slashes
+  #
+  # Usage: URLUtils.join("foo", "bar/", "baz") => "foo/bar/baz"
+  def join(*parts)
```

Comment:
Yeah, as you said, Ruby seems to use `/` for all OSes. So I changed the code to just use `File.join`.